nemo/utils/export_utils.py

```diff
@@ -20,6 +22,134 @@
 
 from nemo.utils import logging
 
+
+class ExportFormat(Enum):
+    """Which format to use when exporting a Neural Module for deployment"""
+
+    ONNX = (1,)
+    TORCHSCRIPT = (2,)
+
+
+_EXT_DICT = {
+    ".pt": ExportFormat.TORCHSCRIPT,
+    ".ts": ExportFormat.TORCHSCRIPT,
+    ".onnx": ExportFormat.ONNX,
+}
+
+
+class BatchNorm1dNoAutoCast(nn.BatchNorm1d):
```

Comment:
How about giving a more general/flexible name for this batch norm as it may get updated and changed over the time. For instance how about naming it something like BatchNorm1dExportable?

Reply:
Many other models in nemo are using BatchNorm1d. Do we know if they have encountered any issues or not or why it just happened for Conformer and not the other models?

Reply:
I had to also use it for CitriNet, yes. 
Regarding name - I think this way it's more descriptive.